source/docs/articles/local/drush-versions.md

```diff
@@ -0,0 +1,45 @@
+keywords: Drupal drush, command line, drupal, terminus drush, cli, site-local drush
+---
+By default, Pantheon runs Drush 8.x on newly created Drupal sites and 5.x on sites created prior to [date]. We highly recommend running Drush 8.x locally when interacting with Drupal sites on Pantheon, but 7.x is also supported. Alias files generated by Pantheon are not compatible with 6.x, however you can execute commands locally with this version by including the `--strict=0` option.
+## Terminus Drush and Local Drush
+[Terminus](/docs/articles/local/cli/) makes remote Drush calls on sites without using a local installation, eliminating compatibility issues between local and remote installs. For more information, see our guide on [Managing Drupal Sites with Terminus and Drush](/docs/guides/terminus-drupal-site-management/).
+## Modifying Default Drush Version
+You can change a site's Drush version on Pantheon by implementing a site-local usage of Drush. The method for doing so differs depending on the current default version being used on the site. Run `drush @pantheon.SITENAME.ENV version` to determine which method below applies to you.
+### Current Default is 8.x
+If the site is currently running Drush 8 on Pantheon and uses composer to manage dependencies, simply add `require drush/drush '8.*'`. If the site does not use composer, commit the desired version of Drush directly to your site's codebase within `__ROOT__/drush`.
+
+This updates the site-local executable path to `__ROOT__/drush/drush/drush`. Check Drush version by running the following:
+```
+$ drush @pantheon.SITENAME.ENV version
+drush version   :  8.0.0-rc2
+```
+### Current Default is 5.x
+For sites running 5.x, create a file called `policy.drush.inc` and place it in the `.drush` folder of your home directory. You can then use hook implementations to modify versions used for remote aliases.
+<div class="alert alert-danger">
+<h4>Warning</h4>
+The Pantheon platform executes hooks as they are written. If a policy file makes a bad choice (e.g. using Drush 7 with Drupal 8), the operation will fail.
+</div>
+The following example uses Drush 8 instead of 5.x for targets on the Pantheon platform. Our `hook_drush_sitealias_alter` function looks like this:
+```php
+function policy_drush_sitealias_alter(&$alias_record) {
+  // Fix pantheon aliases!
+  if ( isset($alias_record['remote-host']) &&
+      (substr($alias_record['remote-host'],0,10) == 'appserver.') ) {
+    $alias_record['path-aliases']['%drush-script'] = 'drush8';
+  }
+}
+```
+Commit the policy file to your site's repository and check Drush version:
+```
+$ drush @pantheon.SITENAME.ENV version
+drush version 5.10.1
+```
+Following the pattern of this example `alter` hook, you can make any adjustments needed to your alias files on the fly.  If you are currently embedding code inside the alias file directly, we highly recommend converting it into a policy file to prepare for the shift to YML in 8.x.
```

Comment:
I don't think many Pantheon customers will be embedding code directly inside the alias file, since the alias files are generally overwritten every time they are downloaded.  I'd just leave that last sentence off, or perhaps add a note that altering is better than modifying, since modifications are erased when new versions of the alias files are downloaded.